gallery/visualtests/nxDateInput.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { Region, Target } = require('@applitools/eyes-webdriverio');
+const { focusTest, focusAndHoverTest, hoverTest, simpleTest } = require('./testUtils');
+
+describe('NxDateInput', function() {
+  const simpleComponentSelector = '#nx-date-input-simple-example .nx-text-input',
+      validatableComponentSelector = '#nx-date-input-validation-example .nx-text-input',
+      disabledComponentSelector = '#nx-date-input-disabled-example .nx-text-input.pristine',
+      disabledValidComponentSelector = '#nx-date-input-disabled-example .nx-text-input.valid',
+      disabledInvalidComponentSelector = '#nx-date-input-disabled-example .nx-text-input.invalid';
+
+  function getInputElementSelector(componentSelector, inputType = 'input') {
+    return `${componentSelector} ${inputType}`;
+  }
+
+  beforeEach(async function() {
+    await browser.url('#/pages/NxDateInput');
+  });
+
+  describe('Simple NxDateInput', function() {
+    it('has a dark border by default', simpleTest(simpleComponentSelector));
+
+    it('has a darker border when hovered',
+        hoverTest(simpleComponentSelector, getInputElementSelector(simpleComponentSelector)));
+    it('has a blue border when focused',
+        focusTest(simpleComponentSelector, getInputElementSelector(simpleComponentSelector)));
+
+    it('has a blue border and blue glow when hovered and focused',
```

Comment:
Minor: this test name, when compared to the one above, makes it sound like the glow is only present on focus+hover

Reply:
> Minor: this test name, when compared to the one above, makes it sound like the glow is only present on focus+hover

Updated the names here: 7400a515b7c87ee66052baacf7ebf419723e8dec